homeassistant/components/homematicip_cloud/services.py

```diff
@@ -32,6 +32,7 @@
 ATTR_CONFIG_OUTPUT_PATH = "config_output_path"
 ATTR_DURATION = "duration"
 ATTR_ENDTIME = "endtime"
+ATTR_VALVE_POSITION = "valve_position"
```

Comment:
Do you need that? Its not referenced somewhere (at least I couldn't find it)

Reply:
Correct, removed